### PR TITLE
Added initial flow support

### PR DESCRIPTION
**Pull request by amilajack**
I've only tested this locally. Would be great if someone else could confirm this before merging

### CODE REVIEW

**Comment by kossnocorp** [comment]
Works at CI: https://travis-ci.org/date-fns/date-fns/builds/181538267#L231 👍 

I going to test, thanks!

**Comment by amilajack** [comment]
Right! Fixed

**Comment by kossnocorp** [comment]
Here too https://github.com/date-fns/date-fns/blob/master/.travis.yml#L22 ;-)

**Comment by kossnocorp** [comment]
It's covered by `npm run lint`

File: package.json [hunk 1]
```diff
@@ -71,11 +72,16 @@
...
     "systemjs-check": "babel-node scripts/systemjs_check.js"
   },
   "standard": {
     "parser": "babel-eslint",
-    "ignore": ["dist/"]
+    "ignore": [
+      "dist/"
+    ],
+    "rules": {
+      "flowtype-errors/show-errors": 2
```
Comment (leshakoss) [inline package.json]: @amilajack @kossnocorp doesn't work for me 🤔 I don't think `standard` allows custom rules

**Comment by leshakoss** [comment]
Closed because `standard` doesn't allow custom rules

**Pull request closed without merging**